Makefile

```diff
@@ -275,14 +278,18 @@ CPPCHECK        = cppcheck $(CSOURCES) --enable=all --platform=unix64 \
 #
 # Things we will build
 #
+TARGET_EXST     = $(BIN_DIR)/$(FORKNAME)_$(FC_VER)_$(TARGET).exst
+TARGET_S19      = $(BIN_DIR)/$(FORKNAME)_$(FC_VER)_$(TARGET).s19
 TARGET_BIN      = $(BIN_DIR)/$(FORKNAME)_$(FC_VER)_$(TARGET).bin
 TARGET_HEX      = $(BIN_DIR)/$(FORKNAME)_$(FC_VER)_$(TARGET).hex
 TARGET_ELF      = $(OBJECT_DIR)/$(FORKNAME)_$(TARGET).elf
+TARGET_EXST_ELF = $(OBJECT_DIR)/$(FORKNAME)_$(TARGET).exst.elf
```

Comment:
since the targets are defined as `TARGET_EXST` via the `.mk` file the resulting filenames already have `_EXST` in them.  e.g. `SPRacingH7EXTREME_EXST.elf`

Reply:
Ok, then naming them `SPRacingH7EXTREME_EXST.exst.elf` seems to be redundant and confusing.

Reply:
the `.exst.elf` has sections patched/replaced, hence the new filename vs the unpatched one. see the use of objcopy around line 337.  the `.exst` binary for distribution is created from the patched .elf using the 'exst' goal.

Reply:
As a suggestion, to make the name of the intermediate elf file without the checksum more indicative, I propose `_nochecksum.elf`.

Reply:
using `_nochecksum.elf` would impact `.elf` generation of other targets, PR updated with alternative solution.